lib/esbuild.ex

```diff
@@ -161,7 +161,10 @@ defmodule Esbuild do
   """
   def run(profile, extra_args) when is_atom(profile) and is_list(extra_args) do
     config = config_for!(profile)
-    args = config[:args] || []
+
+    args =
+      config[:args] ||
+        raise "no args passed to esbuild, check your :esbuild config and make sure you're using the correct profile"
```

Comment:
I actually think we want to check the extra_args and args are not empty. Something like this:

```suggestion
    args = config[:args] || []
    
    if args == [] and extra_args == [] do
      raise "no args passed to esbuild, check your :esbuild config and make sure you're using the correct profile"
    end
```

Then I think fewer tests will break too (or perhaps none).